spec/index.html

```diff
@@ -360,12 +359,23 @@ <h3>Triple Terms and Reification</h3>
           aria-label="An RDF graph containing a reifying triple where the triple term corresponds to an asserted triple"></object>
       </a>
       <figcaption id="fig-asserted-triple-term-alt">
-        An <a>RDF graph</a> containing a <a>reifying triple</a> where the <a>triple term</a> corresponds to an <a>asserted triple</a>.
+        An <a>RDF graph</a> containing a <a>triple annotation</a>, which is a <a>reifying triple</a> whose <a>triple term</a> corresponds to an <a>asserted triple</a>.
         The diagram represents the proposition as a fact using the asserted triple, meaning that the relationship holds.
       </figcaption>
     </figure>
 
-    <p>Note that a <a>triple term</a> may also have another <a>triple term</a> as an <a>object</a>.</p>
+    <p class="note">
+      A <a>proposition</a> denoted by a <a>triple term</a> is an abstract,
+      logical relationship; assertions are always made about some
```

Comment:
I am not happy with this "abstract, logical relationship" because I think that is confusing for readers. Is a Proposition really a relationship? Basically it is a special kind of resource (not happy with that term either). Maybe just "abstract resource"? But I hope that you have a better idea...

Reply:
Triples _are_ relationships between resources, so a proposition is denoted by a triple term, it would seem to me that a relationship is some how involved.

Reply:
We do have the time-honored:
> Asserting an RDF triple says that some relationship, indicated by the predicate, holds between the resources

But I agree that the abstract proposition carrying this meaning is an intricate thing to convey succinctly here. Rather than going further into that, I'll attempt a reformulation.